css/modal.css

```diff
@@ -0,0 +1,205 @@
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1000;
+  padding-top: 100px;
+  padding-bottom: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+/* Modal Content */
+.modal-content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 70%;
+  border-radius: 10px;
+}
+
+/* The Close Button */
+.close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+
```

Comment:
- [ ] Kindly make sure you fix the modal styling in some screen sizes as your Modal Popup is supposed to be responsive. As you can see in the image below the **See Live** and **See Source** buttons are overlapping over some text.
![Screenshot from 2022-09-07 15-33-04](https://user-images.githubusercontent.com/83042742/188892498-1f6101e4-2eaa-4030-940b-5cb8c3f7bc68.png)